### ISSUE TITLE
Support multiple track in video/audio publishing in WebRTC

### ISSUE THREAD

**Issue by mekya** [ant-media/Ant-Media-Server#3188]
**Is your feature request related to a problem? Please describe.**
There are options that user want to publish multiple audio/video tracks at the same time.

**Describe the solution you'd like**
I want to stream multi camera, screen and multi audio at the same time on the WebRTC publisher side

**Comment by peterzanetti** [comment]
Is this feature already a part of latest version? Documents here seem to indicate: https://github.com/ant-media/Ant-Media-Server/wiki/WebRTC-WebSocket-Messaging-Reference

**Comment by mekya** [comment]
No it's not part of it yet. This feature is not yet merged into the master.

**Comment by Madhukaran** [comment]
Hi @burak-58,
     Does this feature is included in Antmedia 2.4.2? I need to publish a camera stream and ScreenShare stream to a room of a same user with Different StreamId's. If Yes, How it is done?

**Comment by Madhukaran** [comment]
> Hi @burak-58, Does this feature is included in Antmedia 2.4.2? I need to publish a camera stream and ScreenShare stream to a room of a same user with Different StreamId's. If Yes, How it is done?

@mekya ?

**Comment by mekya** [comment]
Hi @Madhukaran ,

Yes, just let me explain. This feature is about adding multiple tracks to a single stream. This method is used for playing the WebRTC multi track conference sample. 
https://github.com/ant-media/StreamApp/blob/master/src/main/webapp/multitrack-conference.html

Sending multiple track in a single webrtc session will be added in v2.5.  

Fortunately, I can provide a workaround solution for you.
You can create a conference for a specific room id and you can send one main track(camera stream) and  create  another webrtc session to send the screen track to the same room id.
Basically, it's like one join the conference room with two streams. One is camera and other one is screen share with different webrtc session

I hope it provide some kind of answer. Please let me know if you have further questions
